src/main/java/com/fulcrumgenomics/util/PickIlluminaIndicesCommand.java

```diff
@@ -310,8 +334,9 @@ protected int execute() {
                 log.info("There are " + count + " indices with a minimum edit distance of " + lastMinEdit);
             }
 
-            rankedIndexes.remove(first);
-            if (--count % 100 == 0) log.info("Down to " + count + " indices.");
+            if (rankedIndexes.containsAndRemove(first)) count--;
```

Comment:
I think this can just be `rankedIndices.remove(first)` again given the check of the return value and throwing if false.